test/JDBC/expected/BABEL-3347-vu-verify.out

```diff
@@ -0,0 +1,26 @@
+-- table variable
+SELECT SUBSTRING(text_test, 2,3) FROM babel_3347_table
```

Comment:
The one of the main purposes of version upgrade test is to make sure all objects are correctly migrated. Please create a view for each `SELECT` statement.